router.go

```diff
@@ -243,6 +255,43 @@ func (r *Router) Resource(path string) *Resource {
 	return res
 }
 
+// GetOperation returns the path and
```

Comment:
Missing the end of the sentence? :smile:

Reply:
Addressed in [26405ef](https://github.com/danielgtaylor/huma/pull/82/commits/26405ef6270739c8356d65ee22dcbfbf9ffe69ca)